From: Kurt Roeckx <kurt@roeckx.be>
Date: Mon, 21 Nov 2016 22:15:11 +0100
Subject: Add missing -zdelete for some linux arches

Bug-Debian: https://bugs.debian.org/844715
Applied-Upstream: https://github.com/openssl/openssl/commit/e9a86d6b4951a1c536d5536d1217e6be6455858a
Description: Upstream commit b6d5ba1a9f00 ("Link using -znodelete") forgot to
 update some linux arches.

---
 Configurations/10-main.conf |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Configurations/10-main.conf
+++ b/Configurations/10-main.conf
@@ -702,7 +702,7 @@ sub vms_info {
         cflags           => add("-mabi=ilp32"),
         bn_ops           => "SIXTY_FOUR_BIT RC4_CHAR",
         perlasm_scheme   => "linux64",
-        shared_ldflag    => "-mabi=ilp32",
+        shared_ldflag    => add("-mabi=ilp32"),
     },
 
     "linux-mips32" => {
@@ -767,7 +767,7 @@ sub vms_info {
         cflags           => add("-mx32 -DL_ENDIAN"),
         bn_ops           => "SIXTY_FOUR_BIT",
         perlasm_scheme   => "elf32",
-        shared_ldflag    => "-mx32",
+        shared_ldflag    => add("-mx32"),
         multilib         => "x32",
     },
 
